registry/storage/driver/testsuites/testsuites.go

```diff
@@ -1093,226 +1084,258 @@ func (suite *DriverSuite) TestConcurrentFileStreams(c *check.C) {
 // 	c.Assert(misswrites, check.Not(check.Equals), 1024)
 // }
 
+type DriverBenchmarkSuite struct {
+	Suite *DriverSuite
```

Comment:
```suggestion
	DriverSuite
```
Embedding the struct lets you reference fields of the `DriverSuite` as if they were fields of the `DriverBenchmarkSuite`. E.g.:
```go
func (s *DriverBenchmarkSuite) benchmarkExampleForIllustration(b *testing.B) {
	_, err := s.Suite.StorageDriver.Writer(s.ctx, "foo", false)
	s.Require().NoError(err)
}
```
And you may as well embed by value. We're benchmarking, after all, so the extra pointer indirection adds some noise to the timing.